LayoutTests/fast/events/mouse-cursor-hidden-when-out-of-bounds.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../resources/js-test-pre.js"></script>
```

Comment:
```suggestion
<script src="../../resources/js-test.js"></script>
```

And no need for the -post.js below.